src/components/PlaylistRow.jsx

```diff
@@ -0,0 +1,55 @@
+import React from "react"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
+
+import PlaylistExporter from "./PlaylistExporter"
+
+class PlaylistRow extends React.Component {
+  exportPlaylist() {
+    PlaylistExporter.export(this.props.access_token, this.props.playlist);
+  }
+
+  renderTickCross(condition) {
+    if (condition) {
+      return <FontAwesomeIcon icon={['far', 'check-circle']} size="sm" />
+    } else {
+      return <FontAwesomeIcon icon={['far', 'times-circle']} size="sm" style={{ color: '#ECEBE8' }} />
+    }
+  }
+
+  renderIcon(playlist) {
+    if (playlist.name === 'Liked') {
+      return <FontAwesomeIcon icon={['far', 'heart']} style={{ color: 'red' }} />;
+    } else {
+      return <FontAwesomeIcon icon={['fas', 'music']} />;
+    }
+  }
```

Comment:
if you rely only on 1 property of your `playlist` object, maybe it should be your default parameter

```ts
renderIcon(playlistName) {
  return playlistName === 'Liked' ?
   <FontAwesomeIcon icon={['far', 'heart']} style={{ color: 'red' }} /> : 
   <FontAwesomeIcon icon={['fas', 'music']} />
```